src/main/resources/standardTestCases/multiline_command_line.test

```diff
@@ -0,0 +1,13 @@
+name: multiline_command_line
+testFormat: workflowsuccess
+ignore: true # Fails on JES, Succeeds locally. WTF?
```

Comment:
In fact, different behaviour between `JES` and `Local` if they use the same docker should be a big warning sign!